packages/cli/commands/create.js

```diff
@@ -200,6 +211,62 @@ exports.handler = async options => {
       createTemplate(name, dest, templateType);
       break;
     }
+    case TYPES['api-sample']: {
+      const filePath = path.join(dest, name);
+      if (fs.existsSync(filePath)) {
+        const { overwrite } = await overwriteSamplePrompt(filePath);
+        if (overwrite) {
+          const removingSpinner = ora(`Removing existing ${filePath} folder`);
+          removingSpinner.start();
+          fs.rmdirSync(filePath, { recursive: true });
+          removingSpinner.stop();
+        } else {
+          return;
+        }
+      }
+      const downloadSpinner = ora('Downloading API samples config');
+      downloadSpinner.start();
+      const samplesConfig = await downloadConfig(
+        PROJECT_REPOSITORIES[assetType],
+        'samples.json'
+      );
+      downloadSpinner.stop();
+      if (!samplesConfig) {
+        logger.error(
+          `Currently there are no samples available, please, try again later.`
+        );
+        return;
+      }
+      const { sampleType, sampleLanguage } = await createApiSamplePrompt(
+        samplesConfig
+      );
+      if (!sampleType || !sampleLanguage) {
+        logger.error(
+          `Currently there are no samples available, please, try again later.`
+        );
+        return;
+      }
+      logger.info(
+        `You've chosen ${sampleType} sample written on ${sampleLanguage} language`
+      );
+      commandTrackingContext.templateType = sampleType;
```

Comment:
Since a sample project is pretty different from a template, I don't think we should add this to tracking context.